docs/template/public/kss.less

```diff
@@ -1501,3 +1500,10 @@ body.hyComponentsPage{
 .error-message-table .wide-col {
     min-width: 260px;
 }
+
+.page{

```

Comment:
Please use more specific css classes for the styling of the style guide to avoid conflicts with the components.